### PR TITLE
#2478 SLI evaluation breakdown as barista table (#3023)

### PR DESCRIPTION
**Pull request by johannes-b**
Definition of Done:

- [x]  Table refreshes when clicking on the heatmap
- [x] Fixed size of columns (name 55%, result 10%, value 10%, score 10%, criteria 15%), at least the Name needs more space
- [x]  Column Score shows the weight but should be translated into points between 0 and 100 In the above example, score 1 --> 33.3 points, score 2 --> 66.6 points
- [x]  Split of criteria in warning/pass (currently just the pass criteria are shown)
   - [x]  show warning criteria
- [x] Mark criteria as red (or with an icon as an indicator) when criteria failed
- [x]  Sorting the rows according to the header

### CODE REVIEW

**Comment by Kirdock** [comment]
Also fixes #2475 (the part that is possible)
image

File: cli/internal/cespec/data.go [hunk 1]
```diff
@@ -191,7 +192,7 @@ var evaluationFinishedEventData = keptnv2.EvaluationFinishedEventData{
 				Success: true,
 				Message: "a message",
 			},
-			Targets: []*keptnv2.SLITarget{&keptnv2.SLITarget{
+			PassTargets: []*keptnv2.SLITarget{&keptnv2.SLITarget{
```
Comment (bacherfl) [inline cli/internal/cespec/data.go:195]: please also add an example for the `WarningTargets` property

**Review by bacherfl — CHANGES_REQUESTED** [review]

**Comment by johannes-b** [comment]
A minor change: Can you please align the column "Result" to the right. (to have it the same way like "Value", "pass Criteria", "warning Criteria", and "Score")  

image

**Comment by johannes-b** [comment]
- [ ] The Score column needs to be fixed. Please see my example below, where I have two metrics: response_time (weight = 1), error_rate (weight=2), response_time resulted in a warning (this means that we get half of the points here). In the end, I would like to see: 
- response_time: Score: 16.5
- error_rate Score: 66

In total, this has to sum up to the 83.

Can you please get this resolved? Other than that, the table is a great feature!

image

**Review by ermin-muratovic — DISMISSED** [review]
lgtm

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**